4_predict.R

```diff
@@ -95,15 +95,8 @@ p4_targets_list <- list(
                            by_time = FALSE,
                            num_data_splits = 20
              ),
-             deployment = 'worker'
-  ),
-  
-  # Refresh AWS credentials
-  tar_target(
-    p4_aws_credentials_1,
-    generate_credentials(dummy_var = p4_Boruta_static),
-    deployment = 'main',
-    cue = tar_cue('always')
+             deployment = 'worker',
+             cue = tar_cue(mode = "never")
```

Comment:
I understand why you've taken this approach, but `tar_cue(mode = "never")` does make me a little wary in that it puts onus on the user to understand when parts of the pipeline should be updated or not. Could we add notes somewhere in the documentation about the use of `tar_cue(mode = "never")`? Or do we plan to eventually take these options out and re-run the pipeline in full? Not necessary to address for this PR, just something I wonder about.

Reply:
Yes, I'll edit the README to say which targets are currently set to never update. The critical target to never update for the current work is the `p4_Boruta_static` because that determines which features are used in the models. The model targets could all have the `tar_cue(mode = "never")` removed (it will change model results, but hopefully not by much). I can add an issue to update the pipeline to remove the never cue for model targets. I currently see this as lower priority that can be addressed after the RGCN results are ready.

Reply:
Thanks, Jared. I agree that this is a lower priority and am happy to go with whatever you think is best. I want to avoid a situation where a modeling target doesn't update when we expect it to due to this setting of `tar_cue` but I'm also less familiar with the specifics of the modeling steps and tradeoffs re. computational efficiency.